utopia-remix/app/components/projectActionContextMenu.tsx

```diff
@@ -177,7 +186,10 @@ export const ProjectContextMenu = React.memo(({ project }: { project: ProjectWit
   return (
     <DropdownMenu.Portal>
       <DropdownMenu.Content className={contextMenuDropdown()} align='end' sideOffset={5}>
-        {menuEntries.map((entry, index) => {
+        {menuEntries.filter(Boolean).map((entry, index) => {
```

Comment:
What's the deal with `.filter(Boolean)` ?

Reply:
Yes, I'll try and use that! Thanks

Reply:
<img width="544" alt="image" src="https://github.com/concrete-utopia/utopia/assets/7003853/4e720bec-4ddb-484e-b600-b0656e608df8">
still not convincing typescript :)

Reply:
removed the unneeded `filter(Boolean)`

Reply:
Interesting, so are the `null`s automatically being filtered out elsewhere then?

Reply:
there is a `null` check at the beginning of the mapping function itself (this was the only thing that kept TS quiet).
In any case this UI will be changed with the share popup (no need for the context menu anymore) - hopefully by tomorrow, but if not - soon after

Reply:
this is fixed in #5043